### ISSUE TITLE
Pytorch in hungarian_matching

### ISSUE THREAD

**Issue by benqua** [Visual-Behavior/detr-tensorflow#13]
Hi,

The file [hungarian_matching.py](https://github.com/Visual-Behavior/detr-tensorflow/blob/main/detr_tf/loss/hungarian_matching.py) contains unused references to Pytorch and non-existent functions.

Could this code be removed?
